ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-tests.robot

```diff
@@ -120,3 +152,11 @@ Run Codeflare ODH Test
     IF    ${result.rc} != 0
         FAIL    ${TEST_NAME} failed
     END
+
+Enable Instascale
+    Log To Console    "Enable instascale to true in config map ....."
+    ${result} =    Run Process    oc get cm codeflare-operator-config -n redhat-ods-applications -o yaml | sed -e 's|enabled: false|enabled: true|' | oc apply -f -
```

Comment:
AFAIK the CodeFlare operator needs to be restarted after ConfigMap change, to spin new pod with updated ConfigMap.